primefaces-integration-tests/src/main/resources/META-INF/openwebbeans/openwebbeans.properties

```diff
@@ -0,0 +1,112 @@
+# Copy of https://github.com/apache/openwebbeans/blob/e348f45b8ab3ef825c73dca8a01bb00a0d68e1ad/webbeans-impl/src/main/resources/META-INF/openwebbeans/openwebbeans.properties#L189
+# with /jakarta.faces- added.
+
+######################### Bean Scanning ########################################################
+# A list of known JARs/paths which should not be scanned for beans
+# if they don't have an explicit META-INF/beans.xml
+org.apache.webbeans.scanExclusionPaths=/jre/lib, \
+        /Contents/Home/, \
+        /dt.jar, \
+        /tools.jar, \
+        /bootstrap.jar, \
+        /asm, \
+        /javassist, \
+        /xbean-, \
+        /jconsole.jar, \
+        /geronimo-connector,\
+        /geronimo-j2ee-,\
+        /geronimo-jpa_,\
+        /geronimo-javamail,\
+        /geronimo-transaction,\
+        /commons-, \
+        /arquillian-, \
+        /bsh-, \
+        /shrinkwrap-, \
+        /junit-, \
+        /testng-, \
+        /openjpa-, \
+        /bcel, \
+        /hamcrest, \
+        /mysql-connector, \
+        /testng, \
+        /idea_rt, \
+        /eclipse, \
+        /jcommander, \
+        /tomcat, \
+        /catalina, \
+        /jasper, \
+        /jsp-api, \
+        /myfaces-api, \
+        /myfaces-impl, \
+        /servlet-api, \
+        /javax, \
+        /annotation-api, \
+        /el-api, \
+        /mojarra, \
+        /sisu-guice-, \
+        /sisu-inject-, \
+        /aether-, \
+        /plexus-, \
+        /maven-, \
+        /guava-, \
+        /openwebbeans-, \
+        /bcprov-jdk14-, \
+        /bcmail-jdk14-, \
+        /bctsp-jdk14-, \
+        /bcmail-jdk14-, \
+        /ss_css2-, \
+        /itext-, \
+        /pd4ml-, \
+        /xmlpull-, \
+        /log4j-, \
+        /slf4j-, \
+        /logkit, \
+        /gson-, \
+        /xstream-, \
+        /httpclient-, \
+        /httpcore-, \
+        /backport-util-concurrent-, \
+        /xml-apis, \
+        /xpp3_min-, \
+        /bval-core, \
+        /bval-jsr, \
+        /hsqldb, \
+        /quartz-2, \
+        /jetty-, \
+        /plexus-, \
+        /surefire-, \
+        /byte-buddy-, \
+        /cglib-, \
+        /okhttp-, \
+        /htmlunit-, \
+        /wagon-http-, \
+        /wagon-provider-, \
+        /wagon-file-, \
+        /phantomjsdriver, \
+        /error_prone_annotations-, \
+        /j2objc-, \
+        /xalan-, \
+        /aopalliance-, \
+        /owasp-, \
+        /jdom2-, \
+        /jfreechart-, \
+        /stax-api-, \
+        /jboss-logging-, \
+        /barcode4j-, \
+        /poi-, \
+        /selenium-, \
+        /graphene-, \
+        /httpmime-, \
+        /cssparser-, \
+        /animal-sniffer-annotations-, \
+        /objenesis-, \
+        /xercesImpl-, \
+        /neko-htmlunit-, \
+        /checker-compat-qual-, \
+        /awaitility-, \
+        /okio-, \
+        /jsr305-, \
+        /guice-, \
+        /jsoup-, \
+        /jakarta.faces-
```

Comment:
We have a hack which makes OWB working together with Mojarra (Jakarta Faces) and Tomcat embedded.
Holy ....